docs/release-notes/0.1/release-0.1.md

```diff
@@ -13,7 +13,7 @@ dotnet add package Microsoft.ML
 
 From package manager:
 ```
-Install-Package Microsoft.ML	
```

Comment:
>	 [](start = 28, length = 1)

There was a comment on PR #87 blocking its merge, but since this was depending on that we decided to check it in then quickly fix it here. :P We're very principled.

Reply:
Principled, yes, effective at it, yet to be shown. We came, we saw (the issue open for two days), we pushed.

Reply:
If anything I'd say we're a bit too slow. :D

---
In reply to: [188886371](https://github.com/dotnet/machinelearning/pull/173#discussion_r188886371) [](ancestors = 188886371)